package.json

```diff
@@ -14,30 +14,22 @@
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
   "scripts": {
-    "test": "npm run build && ./node_modules/.bin/gulp --gulpfile gulpfile.js test",
-    "build": "./node_modules/.bin/gulp --gulpfile gulpfile.js build",
-    "watch": "./node_modules/.bin/gulp --gulpfile gulpfile.js watch",
-    "clean": "./node_modules/.bin/gulp --gulpfile gulpfile.js clean",
-    "lint": "./node_modules/.bin/tslint -c tslint.json 'src/**/*.ts'"
-  },
-  "dependencies": {
+    "test": "mocha --opts src/tests/mocha.opts",
+    "build": "rm -rf dist; tsc",
+    "watch": "tsc --watch",
```

Comment:
Makes sense to add the `clean` script, but there is really no need for `./node_modules/.bin/` in scripts.

https://docs.npmjs.com/misc/scripts#path

> If you depend on modules that define executable scripts, like test suites, then those executables will be added to the PATH for executing the scripts. So, if your package.json has this:
> 
>     { "name" : "foo"
>     , "dependencies" : { "bar" : "0.1.x" }
>    , "scripts": { "start" : "bar ./test" } }
> then you could run npm start to execute the bar script, which is exported into the node_modules/.bin directory on npm install.